src/startupscriptgenerator/src/common/buildManifestHelper.go

```diff
@@ -16,7 +16,6 @@ import (
 
 type BuildManifest struct {
 	StartupFileName           string
-	ZipAllOutput              string
```

Comment:
isn't this already merged?

Reply:
No, I missed this and mentioned to @qianz2 that I will be making this change in this PR.